src/mixin/entity-mixin-lit.js

```diff
@@ -18,60 +18,27 @@ export const EntityMixinLit = superclass => class extends superclass {
 			/**
 			 * Entity object that extends the Entity class.
 			 */
-			_entity: { type: Object },
```

Comment:
On the fence about this rename, but this was in response to some of Dave L's comments about not overusing _properties and _methods.

I find them pretty ugly looking and it feels like the entity could be a first class property of entity baed components which could be set directly.

But I could be convinced otherwise of this!

Reply:
It feels odd to me that there would be both an `href` and `entity` on the public API, like there is opportunity for them to get of sync if entity is not read-only to public.

Reply:
OK - i'll switch it back to `_entity`.